db/migrate/20230308160626_create_datasets.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CreateDatasets < ActiveRecord::Migration[7.0]
+  def change
+    create_table :datasets, id: :uuid, &:timestamps
```

Comment:
I almost want to say let's not even add this yet since we aren't doing anything with datasets now...

Reply:
I added it to make the design clearer for the others and to have validations in place that don't permit creating a resource without filling the poly. Just ignore it, if we won't use it atm it will be an empty table, but at least we have the model.